### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Call to a member function getParcelCount() on nul

### ISSUE THREAD

**Issue by Sil-B** [tig-nl/postnl-magento2#41]
### Preconditions
Magento 2.1.10
PostNL 1.2.0

### Steps to reproduce
Try to ship a PostNL (Processing) order in the backend. 

### Expected result
Shipping works as normal. 

### Actual result
Blank page with the text "Colli amount"

### Errlog

```[26-Jan-2018 15:13:30 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getParcelCount() on null in /data/ssd/vhosts/domain.nl/magento2/vendor/tig/postnl-magento2/Block/Adminhtml/Shipment/Options/Create.php:131
Stack trace:
#0 [internal function]: TIG\PostNL\Block\Adminhtml\Shipment\Options\Create->getParcelCount()
#1 /data/ssd/vhosts/domain.nl/magento2/vendor/magento/framework/View/TemplateEngine/Php.php(82): call_user_func_array(Array, Array)
#2 /data/ssd/vhosts/domain.nl/magento2/var/view_preprocessed/html/vendor/tig/postnl-magento2/view/adminhtml/templates/shipment/create/options.phtml(1): Magento\Framework\View\TemplateEngine\Php->__call('getParcelCount', Array)
#3 /data/ssd/vhosts/domain.nl/magento2/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/data/ssd/vhost...')
#4 /data/ssd/vhosts/domain.nl/magento2/vendor/magento/framework/View/Element/Template.php(255): Magento\Framework\View\TemplateEngine\Php->render(Object(TIG\PostNL\Block\Adminhtml\Shipment\Options\Create), '/data/ssd/vhost...', Arr in /data/ssd/vhosts/domain.nl/magento2/vendor/tig/postnl-magento2/Block/Adminhtml/Shipment/Options/Create.php on line 131```

**Comment by tig-jeffreybranderhorst** [comment]
Goodmorning Sil-B,

Do you have a staging website, because we updated our PostNL extension and this error doesn't seem familiar to me.

If the error still exists after the update I would love to investigate this further.  Could you them mail us at servicedesk@tig.nl ?

Best regards, 
Jeffrey Branderhorst.

**Comment by Sil-B** [comment]
@tig-jeffreybranderhorst You've Got Mail

**Comment by tig-dennisvanderhammen** [comment]
For future references, it seems that Sil-B wasn't able to reproduce the error anymore, therefore the investigations were closed. If you have any issues related to this issue feel free to re-open the issue!

**Comment by abvdveen** [comment]
Same issue on live site using Magento 2.2.6, PostNL 1.5.0 

Stack trace:
#0 [internal function]: TIG\PostNL\Block\Adminhtml\Shipment\Options\Create->getParcelCount()
#1 /data/web/magento2/vendor/magento/framework/View/TemplateEngine/Php.php(82): call_user_func_array(Array, Array)
#2 /data/web/magento2/vendor/tig/postnl-magento2/view/adminhtml/templates/shipment/create/options.phtml(47): Magento\Framework\View\TemplateEngine\Php->__call('getParcelCount', Array)
#3 /data/web/magento2/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/data/web/magen...')
#4 /data/web/magento2/vendor/magento/framework/View/Element/Template.php(270): Magento\Framework\View\TemplateEngine\Php->render(Object(TIG\PostNL\Block\Adminhtml\Shipment\Options\Create), '/data/web/magen...', Array)
#5 /data/web/magento2/vendor/magento/framework/View/Element/Template.php(300): Magento\Frame" while reading response header from upstream...

**Comment by tig-dennisvanderhammen** [comment]
Hi @abvdveen and @artemskyi,

We just released version v1.6.0 which fixes a few known extension conflicts that cause this issue. Could you try and see if you can still reproduce this in the latest version?

I also created a backlog item to handle this error a little bit more smoothly, by catching the fatal error.
